elasticsearch/kibana/install.sls

```diff
@@ -0,0 +1,7 @@
+{% from "elasticsearch/kibana/map.jinja" import kibana with context %}
+
+install_kibana:
```

Comment:
This should have a require statement for the package repo being configured. To do that you will need to:
```yaml
include:
  - elasticsearch.pkg_repo (or whatever the proper name is)
```
and then:
```yaml
install_kibana:
  pkg.installed:
    - require:
      - pkgrepo: <id of state to configure Kibana repo>
```